config.go

```diff
@@ -181,6 +182,7 @@ type ticketBuyerOptions struct {
 	NoSpreadTicketPurchases   bool                 `long:"nospreadticketpurchases" description:"Do not spread ticket purchases evenly throughout the window"`
 	DontWaitForTickets        bool                 `long:"dontwaitfortickets" description:"Don't wait until your last round of tickets have entered the blockchain to attempt to purchase more"`
 	VotingAddress             *cfgutil.AddressFlag `long:"votingaddress" description:"Purchase tickets with voting rights assigned to this address"`
+	SplitTx                   uint32               `long:"splittx" description:"Purchase tickets using a split transaction if this value is 1 or number of eligible credits is greater than or equal to this"`
```

Comment:
sorry missed this ealier, this description describes the param better: `The minimum number of ticket purchase inputs that should use a split transaction, options are 1 and 2`